modules/serverless-jaeger-config.adoc

```diff
@@ -16,27 +11,20 @@ To configure Jaeger for use with {ServerlessProductName}, you will need:
 
 .Procedure
 
-. Create and apply a Jaeger custom resource YAML file that contains the following sample YAML:
+. Create and apply a `Jaeger` resource YAML file that contains the following:
 +
-
 .Jaeger custom resource YAML
-+
-
-[source,terminal]
+[source,yaml]
 ----
 apiVersion: jaegertracing.io/v1
 kind: Jaeger
 metadata:
   name: jaeger
   namespace: default
 ----
-
-. Enable tracing for Knative Serving, by editing the `KnativeServing` resource and adding a YAML configuration for tracing.
+. Enable tracing for Knative Serving, by editing the `KnativeServing` resource and adding a YAML configuration for tracing:
```

Comment:
To be consistent with the Jaeger resource above, should this be _`KnativeServing` resouce file_?

Reply:
I'll just change them all to CR I guess